scripts/ci/run_community_csi_test.sh

```diff
@@ -1,16 +1,14 @@
 #!/bin/bash -xe
 
-[ -z "${CONTROLLER_LOGS}" ] && { echo "CONTROLLER_LOGS env is mandatory"; exit 1; }
```

Comment:
why no longer use CONTROLLER_LOGS var? what was its purpose?

Reply:
It forced the job to specify a prefix for the log filename.
I don't think it's necessary, and especially not mandatory.

Reply:
oh, ok